.azure-pipelines/linux-conda-CI.yml

```diff
@@ -19,6 +19,12 @@ jobs:
         onnx.version: 'git'
         onnxrt.version: '==0.2.1'
         sklearn.version: '==0.20.3'
+      Py37-OnnxGit-Nightly:
+        python.version: '3.7'
+        numpy.version: '==1.16.2'
+        onnx.version: 'git'
+        onnxrt.version: 'nightly'
+        sklearn.version: '==0.20.3'
```

Comment:
Make a separate file for just the nightly release check, and then delete all other python build versions from that file. See onnxmltools [linux-CI-nightly file](https://github.com/onnx/onnxmltools/pull/284/files#diff-6989b99615ec0b18248271b18aac06e2) for reference.